internal/executor/doctor/config-jsonschema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "doctor",
+  "description": "Doctor is a ChatGPT integration project that knows how to diagnose Kubernetes problems and suggest solutions.",
+  "type": "object",
+  "properties": {
+    "apiKey": {
+      "description": "OpenAI Secret API Key",
+      "type": "string",
+      "title": "API Key"
+    },
+    "apiBaseUrl": {
+      "description": "OpenAI API Base URL",
+      "type": "string",
+      "title": "API Base URL",
+      "default": "https://api.openai.com/v1"
+    },
+    "defaultEngine": {
+      "description": "Default engine to use",
+      "type": "string",
+      "title": "Default Engine",
+      "default": "text-davinci-003"
+    },
+    "organizationID": {
+      "description": "Optional organization ID",
+      "type": "string",
+      "title": "Organization ID",
+      "default": ""
+    },
+    "userAgent": {
+      "description": "User agent to use for requests",
+      "type": "string",
+      "title": "User Agent",
+      "default": "go-gpt3"
+    },
+    "log": {
```

Comment:
personally, I'm not sure if that should be exposed to the users 🤔 I use it only during development, and sometimes during debugging, so that was the reason why I didn't include it in other plugins  (flux, gh-events) to make them feel like sth easy to configure 😄

Reply:
This is something what we do in most of the plugins, and this is the only way to configure this when using cloud. Yes, we should have consistency here, but I'd rather expose this option to users.